src/Table/Table.tsx

```diff
@@ -932,7 +1032,8 @@ export const Table: React.FunctionComponent<TableProps> = React.memo(
                     selected: row.selected || false,
                     actionsDropdownDropped: row.actionsDropdownDropped || false,
                     expanded: row.expanded || false,
-                    subRows: row.subRows ? getRows(row.subRows) : []
+                    subRows: row.subRows ? getRows(row.subRows) : [],
+                    isEditMode: row.isEditMode || false
```

Comment:
why you need `|| false` here

Reply:
Its initialisation, isEditMode is optional and just like the case is for the remaining objects in the data, we initialized all in getRows .

Reply:
`!!row.expanded` <-- I prefer